lists/in.csv

```diff
@@ -391,7 +391,6 @@ http://www.datedosti.com,DATE,Online Dating,2016-06-14,OONI,
 http://www.deepalaya.org/,HUMR,Human Rights Issues,2016-06-14,OONI,
 http://www.defence.pk/forums/central-south-asia/203273-mim-chief-urges-muslims-help-assam-riots-victims.html,MILX,Terrorism and Militants,2014-04-15,citizenlab,
 http://www.defence.pk/forums/indian-defence/203141-muslims-face-executions-indias-assam-region.html,MILX,Terrorism and Militants,2014-04-15,citizenlab,
-http://www.defence.pk/forums/world-affairs/195202-massarce-muslims-burma-myanmar-\state-rohingyas-community-massacre.html,NEWS,News Media,2014-04-15,citizenlab,
 http://www.defence.pk/forums/world-affairs/195202-massarce-muslims-burma-myanmar-state-rohingyas-community-massacre-2.html,REL,Religion,2014-04-15,citizenlab,
```

Comment:
This is a dupe as the URL with the `-2` suffix redirects to the same page as the one without it.